2-dom-manipulation/form.js

```diff
@@ -0,0 +1,71 @@
+const form = document.querySelector("form");
+const nameField = document.querySelector("#name");
+const lastNameField = document.querySelector("#lastName");
+const emailField = document.querySelector("#email");
+const phoneNumberField = document.querySelector("#phone");
+const radioInputs = document.querySelectorAll("#estrato");
```

Comment:
You shouldn't `queryAll` by `id`. You could use `document.querySelectorAll("[name=estrato]")`

And I would translate "estrato"  to English haha

Reply:
by stratus ???, sorry, copy the file from the previous exercise without the changes 😅